acceptance/tests/external_facts/external_fact_stderr_messages_output_to_stderr.rb

```diff
@@ -11,6 +11,8 @@
 
     if agent['platform'] =~ /windows/
       content = <<EOM
+@ECHO OFF
```

Comment:
What about users that don't have this in their  custom facts files. Is Facter 3 behaving differently based on setting `@ECHO OFF` ?

Reply:
For Facter 3 please see examples in description. If the external facts output in something different than JSON, Yaml or Key=Value, they are not valid.

From documentation (https://puppet.com/docs/puppet/latest/external_facts.html)
```
Custom executable external facts can return data in YAML or JSON format, and Facter parses it into a structured fact. If the returned value is not YAML, Facter falls back to parsing it as a key-value pair.
```